department-of-veterans-affairs/abd-vro
# MCP-1153 Health Check for Lighthouse Service Container

## What was the problem?


There was no health check for lighthouse health api service container.

Associated tickets or Slack threads:

- [MCP-1153](https://amida.atlassian.net/browse/MCP-1153)

## How does this fix it?[^1]


1) Spring Boot actuator/health end point is enabled
2) Dockerfile gets a HEALTH CHECK command identical to app  except the end point (health vs actuator/health)

Tomcat will run for the service from now on.  This was the easiest health check I could find. We can search for a better one later.

## How to test this PR
- Build the application
- Make sure "docker container ls" lists the lighthouse service as healthy


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
